tests/test_camera.py

```diff
@@ -0,0 +1,89 @@
+import os
+import unittest
+from unittest.mock import patch
+
+import cv2
+
+import sense.camera as camera
+
+cwd = os.getcwd()
+video_path = os.path.join(cwd, 'tests/resources/test_video.mp4')
+
+
+class TestVideoSource(unittest.TestCase):
+
+    def setUp(self) -> None:
+        self.video = camera.VideoSource(filename=video_path)
+
+    @patch('sense.camera.VideoSource.pad_to_square')
+    def test_get_image(self, mock_pad_to_square):
+        square_img_path = os.path.join(cwd, 'tests/resources/square_SENSE.png')
```

Comment:
Paths should be system-independent if possible and `os.path.join` is already the right tool for that:
```python
square_img_path = os.path.join(cwd, 'tests', 'resources', 'square_SENSE.png')
```